airbyte-integrations/connectors/destination-iceberg-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/iceberg/v2/IcebergV2DataDumper.kt

```diff
@@ -73,7 +73,7 @@ object IcebergV2DataDumper : DestinationDataDumper {
                     AirbyteRecordMessageMetaChange.Reason.fromValue(
                         change.getField("reason") as String
                     )
-                DestinationRecord.Change(field, changeValue, reason)
+                Meta.Change(field, changeValue, reason)
```

Comment:
blegh. hopefully you didn't spend too long on this diff - I basically rewrote this file in another PR https://github.com/airbytehq/airbyte/pull/49848/files#diff-dea5bd216b2112537b1a7a4204538cbe8170e41edf15363dd03c7034ec462266